src/components/fields/error-field/types.ts

```diff
@@ -8,10 +7,8 @@ export interface IErrorFieldValidationRule {
 	errorMessage?: string | undefined;
 }
 
-type TErrorFieldChildren = IAlertSchema | ITextSchema | IWrapperSchema | IOrderedListSchema | IUnorderedListSchema;
-
 export interface IErrorFieldSchema
 	extends Pick<IBaseFieldSchema<"error-field", undefined, IErrorFieldValidationRule>, "showIf" | "uiType"> {
 	validation?: [IErrorFieldValidationRule] | undefined;
-	children?: Record<string, TErrorFieldChildren> | undefined;
+	children?: Record<string, TBlockElementSchema> | undefined;
```

Comment:
can we exclude `IGridSchema` and `TWrapperSchema` but add `TInlineElementSchema`?

Reply:
would grid be more appropriate as a container element then? I view it as a lower-level layout helper so it's the same usage as `div` for me

Reply:
yeah it makes more sense but if we consider grid a container element, wrapper should be a container element as well. the implication is on the review component, it will have to be updated to accept the wrapper schema

Reply:
since wrapper is not clearly defined yet, I'll exclude it from the block elements and let components opt in

moving grid to container since it's based on the DS Layout, which is meant for top level content anyway